mesonbuild/scripts/yelphelper.py

```diff
@@ -29,6 +29,22 @@
 parser.add_argument('--langs', dest='langs', default='')
 parser.add_argument('--symlinks', type=bool, dest='symlinks', default=False)
 
+def read_linguas(src_sub):
```

Comment:
Would `from .gettext import read_linguas` be enough? Or should I move it to `__init__.py` and import it in both gettxt.py and yelphelper.py?